lua/luasnip/extras/fmt.lua

```diff
@@ -97,7 +100,18 @@ local function interpolate(fmt, args, opts)
 		-- The nodes are modified in-place as part of constructing the snippet,
 		-- modifying one node twice will lead to UB.
 		if used_keys[key] then
-			table.insert(elements, copy3(args[key]))
+			local jump_index = nil -- For nodes that don't have a jump index, copy it instead
+			if
+				type(args[key]) == "table"
+				and args[key].get_jump_index ~= nil
+			then
+				jump_index = args[key]:get_jump_index()
+			end
```

Comment:
Wait, this isn't supposed to be here..

Reply:
yeah, i messed up when i rearranged the commits, sorry

Reply:
ok, I hope there's no more problems

Reply:
Okay :D